build.assets/tooling/cmd/eventtype-mapping/main.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2023 Gravitational, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"regexp"
+	"text/template"
+
+	"github.com/Masterminds/sprig/v3"
+	"github.com/gravitational/trace"
+)
+
+var ignoredEventTypes = map[string]struct{}{
+	"device": {},
+}
+
+func main() {
+	if len(os.Args) != 2 {
+		log.Fatal(trace.BadParameter("the executable must be called with a single argument: the path of the events/api.go file."))
+	}
+	filePath := os.Args[1]
+	mapping := iterateOverFile(filePath, getMappingEntry)
+	output, err := dumpMapping(mapping)
+	if err != nil {
+		log.Fatal(err)
+	}
+	// TODO: run fmt before outputting
+	fmt.Println(string(output))
+}
+
+var reString = `\s*([^ ]+)Event\s+=\s+"([^ ]+)"`
```

Comment:
Can you add a few examples of what this regex matches? The variable name is also not self-explanatory.